src/main/java/org/jenkinsci/plugins/docker/workflow/client/LaunchResult.java

```diff
@@ -31,12 +34,19 @@
 public class LaunchResult {
 
     private int status;
-    private String out;
-    private String err;
+    private @Nonnull String out;
+    private @Nonnull String err;
 
     LaunchResult() {
+        this(0, "", "");
     }
 
+    public LaunchResult(int status, String out, String err) {
```

Comment:
Yes, need annotations.
String::trim() trims space-only strings to empty strings, so there should not be a NPE